src/glang/codegen/builtin_types.py

```diff
@@ -271,18 +272,18 @@ def are_equivalent(self, other: TypeDefinition) -> bool:
 
     @property
     def size(self) -> int:
-        assert False
+        raise AssertionError
```

Comment:
Huh, why isn't this instantiating the AssertionError? Is this not a class?

Reply:
> Otherwise, [raise](https://docs.python.org/3/reference/simple_stmts.html#raise) evaluates the first expression as the exception object. It must be either a subclass or an instance of [BaseException](https://docs.python.org/3/library/exceptions.html#BaseException). If it is a class, the exception instance will be obtained when needed by instantiating the class with no arguments.

Oh god. This is jank

Reply:
Yeah, I can't believe this is a feature...